src/js/components/Page/PageContext.js

```diff
@@ -0,0 +1,3 @@
+import { createContext } from 'react';
+
+export const PageContext = createContext({});
```

Comment:
```suggestion
export const PageContext = createContext({});

```

Reply:
need new line at end of file i think

Reply:
Looks like lint is removing the extra line when I try to add it

Reply:
Fixed